includes/Modules/Search_Console.php

```diff
@@ -538,12 +540,12 @@ protected function setup_info() {
 	}
 
 	/**
-	 * Get the configured Webmasters service instance.
+	 * Get the configured SearchConsole service instance.
```

Comment:
While we're at it, let's add a `since` annotation here, which was previously missing. Since it's a new method though, we can use the `n.e.x.t` tag.
```suggestion
	 * Get the configured SearchConsole service instance.
	 *
	 * @since n.e.x.t
```